### PR TITLE
Add server level policy for vulnerability assessment

### PR DESCRIPTION
**Pull request by talhers**
As part of the ATP concept, we would like to add a new API to manage vulnerability assessment policy at server level, for servers and managed instances.

These APIs are similar to the regular Vulnerability Assessment APIs on a database, with the difference that they don't contain "/databases/{database name}/"

This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by talhers** [comment]
@yaakoviyun , @yoavfr , @ggildin

File: specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/ManagedInstanceVulnerabilityAssessments.json [hunk 1]
```diff
@@ -0,0 +1,378 @@
...
+        "storageContainerSasKey": {
+          "description": "A shared access signature (SAS Key) that has write access to the blob container specified in 'storageContainerPath' parameter. If 'storageAccountAccessKey' isn't specified, StorageContainerSasKey is required.",
+          "type": "string",
+          "x-ms-mutability": [
+            "create",
+            "update"
+          ]
+        },
+        "storageAccountAccessKey": {
+          "description": "Specifies the identifier key of the vulnerability assessment storage account. If 'StorageContainerSasKey' isn't specified, storageAccountAccessKey is required.",
```
Comment (yaakoviyun) [inline specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/ManagedInstanceVulnerabilityAssessments.json]: seems confusing.
there is no "vulnerability assessment storage account".

Maybe use: "storage account for vulnerability assessment scan results"?

**Review by jianghaolu — APPROVED** [review]

**Comment by yaakoviyun** [comment]
@ravbhatnagar  - can you please review?

**Comment by jhendrixMSFT** [comment]
@AutorestCI regenerate azure-sdk-for-go

**Comment by talhers** [comment]
@jianghaolu Please merge
Thanks!

**Comment by talhers** [comment]
There is a newer version already merged, this PR can be closed

**Pull request closed without merging**